fiaas_skipper/deploy/deploy.py

```diff
@@ -136,13 +136,14 @@ def _get_status(dep, app):
 
 
 class StatusTracker(Thread):
-    def __init__(self, cluster, application):
+    def __init__(self, cluster, application, interval=STATUS_UPDATE_INTERVAL):
```

Comment:
The production code never uses the default here, so maybe just remove it to makenit clearer?